lib/definition.js

```diff
@@ -71,6 +99,8 @@ module.exports = class Definition {
     this._match = null
     this._min = null
     this._max = null
+    this._list_separator = /\s{1}|,/
```

Comment:
nitty, but the `{1}` isn't really needed.  The braces are good if you need an upper and lower bound, but to match just 1, you only need `\s`. I would suggest changing it to `+` though (don't forget the docs too) so that multiple spaces become 1 thing, and it'll be less to filter out.
```suggestion
    this._list_separator = /\s+|,/
```
```bash
> re = /\s|,/
/\s|,/
> 'one, two,     three'.split(re)
[
  'one',   '',
  'two',   '',
  '',      '',
  '',      '',
  'three'
]
```
VERSUS
```bash
> re = /\s+|,/
/\s+|,/
> 'one, two,     three'.split(re)
[ 'one', '', 'two', '', 'three' ]
```